public/style.css

```diff
@@ -15,3 +28,57 @@
 ul {
   list-style: none;
 }
+
+#list-of-rules{
+  list-style:square;
+  text-align: left;
+  margin-left: 20px;
+  margin-right: 40px;
+  background-color: #DBE2EF;
+}
+
+#list-of-rules li{
+  background-color: #DBE2EF;
+  line-height: 1.5em;
+}
+
+#start-quiz-button {
+  background-color: #112D4E;
+  border-color: #112D4E;
+  color: #F9F7F7;
+  width: auto;
+  height: auto;
```

Comment:
These two properties might not be needed because default value of width and height are already `auto`.